scripts/verifyExamples.sh

```diff
@@ -21,4 +21,7 @@ set -o pipefail
 mdrip -alsologtostderr -v 10 --blockTimeOut 20m0s --mode test \
     --label verifyExamples site/reference
 
+mdrip -alsologtostderr -v 10 --blockTimeOut 20m0s --mode test \
+    --label verifyGuides site/book
```

Comment:
verifyBook?

Reply:
I will change the name to verifyBook and commit them.